applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpurad/images/ipc_radio/prj_release.conf

```diff
@@ -4,7 +4,6 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-CONFIG_LOG=n
```

Comment:
I would leave explicitly CONFIG_LOG=n

Reply:
Not needed. `CONFIG_LOG` is disabled by default.